### DISCUSSION TITLE
github contribution open source

### DISCUSSION THREAD

**Discussion by Madhurchandran** [community/community#36288] [New to GitHub]
New to GitHub, how to get started with 1st contribution, I have basic knowledge of python.
How to use the knowledge contribute at some projects gain some experience?

**Comment by radar-blip** [comment]
Create a Repository (include a README) while creating it. Edit the README file and click on the Commit button at the bottom.
That will get you started!

Here is some more info: [About Commits](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/about-commits)

**Reply by Madhurchandran** [reply]
thank you

**Comment by haseeb-xd** [comment]
There are so many websites where you can find projects (repositories) for contribution. 

For example:
Go to this website https://goodfirstissues.com/ , and filter with  Language: `python` . You will find issues which are beginner friendly.

These are some of the website where you can find beginner friendly issues:

1. https://www.firsttimersonly.com/
2. https://goodfirstissues.com/
3. https://goodfirstissue.dev/

you will find many of these websites out there.

**Reply by Madhurchandran** [reply]
thank you

**Comment by ossanna16** [comment]
👋 Welcome to the GitHub Community on Discussions, @Madhurchandran! Happy to have you here 🙂
To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204). To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912).

**Reply by Madhurchandran** [reply]
thank you

**Comment by CBID2** [comment]
Hi @Madhurchandran! I highly recommend joining this [organization](https://github.com/EddieHubCommunity?view_as=public#eddiehub---collaboration-1st-code-2nd)

**Comment by melvincwng** [comment]
Hey @Madhurchandran, I recommend joining this beginner friendly open source organization over [here](https://github.com/Your-First-Open-Source-Project/Your-First-Open-Source-Project).

You can refer to the README.md for more information :)
